app/models/lettings_log.rb

```diff
@@ -435,13 +435,29 @@ def self.to_csv(user = nil)
     Csv::LettingsLogCsvService.new(user).to_csv
   end
 
+  def validation_beds
```

Comment:
Is `beds` only set when the log is a general needs letting?

Reply:
Yes - we don't ask a question about beds for supported housing logs, and this corresponds to the rows in the rent range data with beds = 0

Reply:
Would `beds` not be zero already, in that case?

Reply:
Nope - just checked, `beds` is nil for supported housing